tests/unit_tests/docmaps/v2/codecs/elife_manuscript_test.py

```diff
@@ -73,6 +75,25 @@ def test_should_populate_docmaps_elife_manuscript_output(self):
         }
 
 
+class TestGetDocmapElifeManuscriptOutputForPublished:
+    def test_should_populate_docmaps_elife_manuscript_output_for_published(self):
+        result = get_docmap_elife_manuscript_output_for_published(
+            query_result_item=DOCMAPS_QUERY_RESULT_ITEM_1,
+            manuscript_version=MANUSCRIPT_VERSION_1
+        )
+        assert result == {
+            'type': 'preprint',
+            'published': RP_PUBLICATION_TIMESTAMP_1,
```

Comment:
Maybe it would be good to refer to `MANUSCRIPT_VERSION_1` to make the link more obvious.

Reply:
I think somewhere we need to parse the date + UTC time from the Google Sheet to a timestamp, and format it again. We shouldn't just pass through the string assuming it is valid.

We could parse it in SQL or Python.